<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWidget</class>
 <widget class="QFrame" name="ColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="slidersLayout">
     <item>
      <widget class="QSlider" name="rSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"> QSlider::groove:vertical {
     background: #cc0000;
     position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
     left: 4px; right: 4px;
 }

 QSlider::handle:vertical {
     height: 10px;
     background: #cc0000;
     margin: 0 -4px; /* expand outside the groove */
 }

 QSlider::add-page:vertical {
     background: #ef2929;
 }

 QSlider::sub-page:vertical {
     background: #2e3436;
 }</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>127</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="gSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"> QSlider::groove:vertical {
     background: #4e9a06;
     position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
     left: 4px; right: 4px;
 }

 QSlider::handle:vertical {
     height: 10px;
     background: #4e9a06;
     margin: 0 -4px; /* expand outside the groove */
 }

 QSlider::add-page:vertical {
     background: #73d216;
 }

 QSlider::sub-page:vertical {
     background: #2e3436;
 }</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>127</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="bSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"> QSlider::groove:vertical {
     background: #204a87;
     position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
     left: 4px; right: 4px;
 }

 QSlider::handle:vertical {
     height: 10px;
     background: #204a87;
     margin: 0 -4px; /* expand outside the groove */
 }

 QSlider::add-page:vertical {
     background: #3465a4;
 }

 QSlider::sub-page:vertical {
     background: #2e3436;
 }</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>127</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #7f7f7f; border: 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
